src/bundle/Resources/public/js/scripts/helpers/react.helper.js

```diff
@@ -0,0 +1,24 @@
+const createDynamicRoot = (contextDOMElement = window.document.body, id) => {
```

Comment:
`window.document.body ` you can get this object by function `getRootDOMElement` from https://github.com/ibexa/admin-ui/blob/main/src/bundle/Resources/public/js/scripts/helpers/context.helper.js. We will avoid possible problems in the future if this code is used outside DXP